container-images/mds-agency/package.json

```diff
@@ -10,7 +10,6 @@
   "scripts": {
     "bundle": "webpack --mode=production --env.npm_package_name=${npm_package_name} --env.npm_package_version=${npm_package_version}",
     "image": "yarn bundle && ../../bin/mdsctl -c image-name=mds-agency:${npm_package_version} buildImage",
-    "start": "yarn watch server",
```

Comment:
No more `yarn watch server`? Is that being consolidated under a different command, or is this a one-off change for `Agency`?

Reply:
It was only removed from the docker image packages. We don’t use it there. It’s still available when running packages locally.

Reply:
It was only removed from `container-images/mds-agency` not `packages/mds-agency`